lib/cli/init.js

```diff
@@ -0,0 +1,11 @@
+const init = require('../model/init');
```

Comment:
In the architecture, we have a clean separation between input/ and output/.

index.js calls caporal-based CLI controller structure put in place in lib/controller module. The controller module would call the relevant parts of the lib/cli/input module. The controller receives input objects, hands them over to model objects available in lib/model, receives outputs and passes them over to lib/cli/output.